tests/AP1roG/AP1roGGeminalCoefficients_test.cpp

```diff
@@ -0,0 +1,58 @@
+#define BOOST_TEST_MODULE "AP1roGGeminalCoefficients"
+
+
+#include "AP1roG/AP1roGGeminalCoefficients.hpp"
+
+#include <boost/test/unit_test.hpp>
+#include <boost/test/included/unit_test.hpp>  // include this to get main(), otherwise the compiler will complain
+
+
+BOOST_AUTO_TEST_CASE ( default_constructor ) {
+    GQCG::AP1roGGeminalCoefficients gem_coeff;
+}
+
+
+BOOST_AUTO_TEST_CASE ( constructor ) {
+
+    // Check a correct constructor
+    GQCG::AP1roGGeminalCoefficients g (4, 6);
+
+    // We can't create 4 geminals in 4 orbitals
+    BOOST_CHECK_THROW(GQCG::AP1roGGeminalCoefficients(4, 4), std::invalid_argument);
+}
+
+
+BOOST_AUTO_TEST_CASE ( constructor_vector ) {
+
+    // Check a correct constructor
+    Eigen::VectorXd g = Eigen::VectorXd::Zero(6);
+    BOOST_CHECK_NO_THROW(GQCG::AP1roGGeminalCoefficients(g, 2, 5));
```

Comment:
GQCG::AP1roGGeminalCoefficients(g, 2, 5) should this be 
GQCG::AP1roGGeminalCoefficients (g, 2, 5)?